<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="jest tests" tests="5" failures="1" errors="0" time="9.218">
  <testsuite name="GET /api/cartItem/cart-items/:id" errors="0" failures="0" skipped="0" timestamp="2024-09-30T20:18:31" time="8.617" tests="3">
    <testcase classname="GET /api/cartItem/cart-items/:id 200" name="GET /api/cartItem/cart-items/:id 200" time="0.05">
    </testcase>
    <testcase classname="GET /api/cartItem/cart-items/:id 404" name="GET /api/cartItem/cart-items/:id 404" time="0.016">
    </testcase>
    <testcase classname="GET /api/cartItem/cart-items/:id 500" name="GET /api/cartItem/cart-items/:id 500" time="0.008">
    </testcase>
  </testsuite>
  <testsuite name="GET /api/orderItem/order-items/:id" errors="0" failures="1" skipped="0" timestamp="2024-09-30T20:18:31" time="8.912" tests="2">
    <testcase classname="GET /api/orderItem/order-items/:id 404" name="GET /api/orderItem/order-items/:id 404" time="0.041">
    </testcase>
    <testcase classname="GET /api/orderItem/order-items/:id 200" name="GET /api/orderItem/order-items/:id 200" time="0.022">
      <failure>Error: expect(received).toBe(expected) // Object.is equality

Expected: 200
Received: 404
    at toBe (/app/test/controllers/orderItemes/item.getId.test.js:41:33)
    at call (/app/test/controllers/orderItemes/item.getId.test.js:2:1)
    at Generator.tryCatch (/app/test/controllers/orderItemes/item.getId.test.js:2:1)
    at Generator._invoke [as next] (/app/test/controllers/orderItemes/item.getId.test.js:2:1)
    at asyncGeneratorStep (/app/test/controllers/orderItemes/item.getId.test.js:2:1)
    at asyncGeneratorStep (/app/test/controllers/orderItemes/item.getId.test.js:2:1)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)</failure>
    </testcase>
  </testsuite>
</testsuites>